### PR TITLE
Added Death Battletome: Nighthaunt data

### PR DESCRIPTION
**Pull request by CptAsgard**

### CODE REVIEW

File: src/army/nighthaunt/abilities.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  {
+    name: `Wave of Terror`,
+    desc: `If you make an unmodified charge roll of 10+ for a friendly NIGHTHAUNT unit, it can fight immediately after you complete the charge move. This does not stop the unit from being picked to fight in the combat phase of the same turn.`,
+    when: [CHARGE_PHASE],
+  },
+  {
+    name: `COMMAND ABILITY: Spectral Summons`,
+    desc: `You can use this command ability at the start of your movement phase. If you do so, pick a friendly NIGHTHAUNT unit that is on the battlefield. Remove that unit from the battlefield, and then set it up wholly within 12" of your general and more than 9" from any enemy models. This counts as their move for that movement phase.`,
+    when: [START_OF_MOVEMENT_PHASE],
```
Comment (daviseford) [inline src/army/nighthaunt/abilities.ts]: remove the `COMMAND ABILITY: ` part of the string and instead add `command: true` to this object

File: src/army/nighthaunt/units.ts [hunk 2]
```diff
@@ -0,0 +1,646 @@
...
+        desc: `Add 1 to the number of models that flee from enemy units that fail a battleshock test while they are within 12" of this model.`,
+        when: [BATTLESHOCK_PHASE],
+      },
+      {
+        name: `Wail of the Damned`,
+        desc: `At the start of your shooting phase, roll 2D6 for each enemy unit within 10" of this model. If the roll for the unit is higher than its Bravery characteristic, it suffers D3 mortal wounds.`,
+        when: [START_OF_SHOOTING_PHASE],
+      },
+      {
+        name: `COMMAND ABILITY: No Rest For the Wicked`,
```
Comment (daviseford) [inline src/army/nighthaunt/units.ts]: remove the `COMMAND ABILITY: ` part of the string and instead add `command: true` to this object

File: src/army/nighthaunt/units.ts [hunk 3]
```diff
@@ -0,0 +1,646 @@
...
+        desc: `Ignore modifiers (positive or negative) when making save rolls for attacks that target this model.`,
+        when: [COMBAT_PHASE],
+      },
+      {
+        name: `Stolen Hours`,
+        desc: `Each time a wound inflicted by this model's Sword of Stolen Hours slays an enemy HERO, heal 1 wound allocated to this model.`,
+        when: [COMBAT_PHASE],
+      },
+      {
+        name: `COMMAND ABILITY: Spectral Overseer`,
```
Comment (daviseford) [inline src/army/nighthaunt/units.ts]: remove the `COMMAND ABILITY: ` part of the string and instead add `command: true` to this object

File: src/army/nighthaunt/units.ts [hunk 4]
```diff
@@ -0,0 +1,646 @@
...
+        desc: `Ignore modifiers (positive or negative) when making save rolls for attacks that target this model.`,
+        when: [COMBAT_PHASE],
+      },
+      {
+        name: `Stolen Hours`,
+        desc: `Each time a wound inflicted by this model's Sword of Stolen Hours slays an enemy HERO, heal 1 wound allocated to this model.`,
+        when: [COMBAT_PHASE],
+      },
+      {
+        name: `COMMAND ABILITY: Lord of Gheists`,
```
Comment (daviseford) [inline src/army/nighthaunt/units.ts]: remove the `COMMAND ABILITY: ` part of the string and instead add `command: true` to this object

**Comment by CptAsgard** [comment]
> 🔥 Love it! Really nice work. Did you find the codebase easy to work with? I'm going to commit a couple of minor changes and then merge/upload this bad boy!

Good stuff! Yeah I think you did a good job making the data easy to extend. The README was clear as well, although I had to run 'npm install' in the folder first which was unclear.

**Comment by daviseford** [comment]
Gotcha! I just added a line to the README about running `yarn install` - thanks. Any chance you'd be interested in adding more armies? I can always use the help :)

**Comment by CptAsgard** [comment]
> Gotcha! I just added a line to the README about running `yarn install` - thanks. Any chance you'd be interested in adding more armies? I can always use the help :)

If I get a new battletome I'll definitely add it

**Pull request merged**